### PR TITLE
Remove libpod.conf from repo

### PR DESCRIPTION
**Pull request by mheon**
Now that we're shipping containers.conf, we don't want to provide a libpod.conf anymore. This removes libpod.conf from the repo and as many direct uses as I can find.

There are a few more mentions in the documentation, but someone more familiar with containers.conf should make those edits.

### CODE REVIEW

**Comment by mheon** [comment]
They may cause CI to explode. Will deal with it if it happens.

**Comment by baude** [comment]
yeah LGTM

**Comment by TomSweeneyRedHat** [comment]
@lsm5 PTAL

**Comment by mheon** [comment]
I think this will go on our master CI without much of an issue. I think it will actively explode on the 1.9 branch, so I don't think this can be backported there.

File: libpod.conf [hunk 1]
```diff
@@ -1,181 +0,0 @@
-# libpod.conf is the default configuration file for all tools using libpod to
```
Comment (TomSweeneyRedHat) [inline libpod.conf:1]: Should we keep the file, but just have it contain a small comment/note that its been removed as of version 1.9 and replaced by containers.conf?

**Comment by lsm5** [comment]
LGTM, I might need to adjust packaging but no biggie..

**Comment by rhatdan** [comment]
/lgtm
/hold
I don't think we should leave the file around.

**Pull request merged**